src/Spout/Writer/XLSX/Internal/Worksheet.php

```diff
@@ -212,8 +212,10 @@ private function getCellXML($rowIndex, $cellNumber, $cellValue, $styleId)
         $columnIndex = CellHelper::getCellIndexFromColumnIndex($cellNumber);
         $cellXML = '<c r="' . $columnIndex . $rowIndex . '"';
         $cellXML .= ' s="' . $styleId . '"';
-
-        if (CellHelper::isNonEmptyString($cellValue)) {
+        
+        if (CellHelper::isFormulaString($cellValue)) {
+            $cellXML .= '><f>'.substr($cellValue,1).'</f><v>0</v></c>'; // seriously, that's it.
```

Comment:
Tested on OSX, with excel 15 and numbers.

<v>0</v> works because excel formulas are recalculated when an excel file is loaded.

You are correct that _reading_ a formula will result in unexpected results, I had hoped the title of the PR was specific enough for this to be clear.

Thanks for coming back to me, glad to have been of any use at all.